src/Components/Aspire.Azure.AI.Inference/ChatCompletionsClientSettings.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using Aspire.Azure.Common;
+using Azure.Core;
+using Microsoft.Identity.Client.Platforms.Features.DesktopOs.Kerberos;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Represents configuration settings for Azure AI Chat Completions client.
+/// </summary>
+public sealed class ChatCompletionsClientSettings : IConnectionStringSettings
+{
+    private bool? _disableTracing;
+
+    /// <summary>
+    /// Gets or sets the connection string used to connect to the AI Foundry account.
+    /// </summary>
+    /// <remarks>
+    /// If <see cref="ConnectionString"/> is set, it overrides <see cref="Endpoint"/> and <see cref="Credential"/>.
```

Comment:
This comment is a bit misleading, since setting ConnectionString will populate Endpoint in ParseConnectionString below.

Reply:
I think the comment was lifted from here: https://github.com/dotnet/aspire/blob/38591121f7667cc5c6b9d7e17cd086d4909c1d6c/src/Components/Aspire.Azure.Data.Tables/AzureDataTablesSettings.cs#L18